src/app/public/modules/colorpicker/colorpicker.component.scss

```diff
@@ -34,6 +34,14 @@
   color: transparent;
 }
 
+::ng-deep {
```

Comment:
Without a preceding class this could overwrite all popovers all the page correct? Should we qualify this with a preceding class or element?

Reply:
Yikes, good catch.

Reply:
LGTM